data/usa_zip_codes_v7.geo.json

```diff
@@ -292743,7 +292743,7 @@
   "type": "Feature",
   "geometry": {
     "type": "Polygon",
-    "coordinates": [[[-110.978197, 32.207113], [-111.132337, 32.221238], [-111.132337, 32.220454], [-111.071089, 32.171013], [-110.991467, 32.172583], [-110.967989, 32.178076], [-110.966968, 32.182785], [-110.961864, 32.181215], [-110.960843, 32.178076], [-110.908783, 32.185139], [-110.909804, 32.196126], [-110.910824, 32.207113], [-110.935323, 32.207113], [-110.947573, 32.207113], [-110.960843, 32.207113], [-110.978197, 32.207113]], [[-110.947573, 32.207113], [-110.94349, 32.205543], [-110.944511, 32.202404], [-110.947573, 32.207113]]]
```

Comment:
There is an unnecessary vertex removed and then the coordinates of the interior ring are moved slightly 

![image](https://github.com/elastic/ems-file-service/assets/188264/1afe60f5-3dbd-463d-bccd-1a22cf422259)